diesel_cli/tests/print_schema/print_schema_custom_types_overriding_derives_works/diesel.toml

```diff
@@ -0,0 +1,4 @@
+[print_schema]
+file = "src/schema.rs"
+with_docs = true
+custom_type_derives = ["diesel::sql_types::SqlType", "core::fmt::Debug"]
```

Comment:
> I'm unsure about the tests though: they pass, and I only had to update the `diesel.toml` files and not the CLI args, so I'm confused as to how tests pass.

Isn't it that they use both the corresponding diesel.toml and CLI args? (So in order to see that CLI args work we could remove these and see whether the args specified at https://github.com/diesel-rs/diesel/pull/3308/files#diff-4f7007ed0625d981d1a3230723b3a57050fe8909d83510011656a2693c5304afR225-R227 hit in such a way that the output doesn't change?)

Reply:
> Isn't it that they use both the corresponding diesel.toml and CLI args? (So in order to see that CLI args work we could remove these and see whether the args specified at [#3308 (files)](https://github.com/diesel-rs/diesel/pull/3308/files#diff-4f7007ed0625d981d1a3230723b3a57050fe8909d83510011656a2693c5304afR225-R227) hit in such a way that the output doesn't change?)

I was able to trace and figure out how the tests are run: first the CLI args are used as configuration for generating the schema, and if it generates the expected output, the configuration from the corresponding `diesel.toml` is used for generating the schema. When I remove this line from the `diesel.toml` file but keep the CLI args as is:

```toml
[print_schema]
# ...
custom_type_derives = ["diesel::sql_types::SqlType", "core::fmt::Debug"]
```

the test as a whole fails (as expected), since using the CLI args as config generates the expected output, but using the `diesel.toml` file doesn't include `Debug` in the list of derives. So, please ignore my (outdated) comment about me not being able to understand how the tests passed.

@weiznich This test verifies that overrides work for both the config file and CLI args. Please let me know if you'd like me to include a test for only the CLI args, or if any additional tests are necessary. I will be including a changelog entry soon.

Reply:
:+1: That's fine for me. We don't need an additional test then.